mixpanel_oauth_to_redshift.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"crypto/md5"
+	"database/sql"
+	"encoding/hex"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"sort"
+	"time"
+
+	"github.com/Clever/pathio"
+	_ "github.com/lib/pq" // Postgres Driver
+)
+
+var (
+	aws_access_key_id     = os.Getenv("AWS_ACCESS_KEY_ID")
+	aws_secret_access_key = os.Getenv("AWS_SECRET_ACCESS_KEY")
+	aws_region            = os.Getenv("AWS_REGION")
+	mixpanel_api_key      = os.Getenv("MIXPANEL_API_KEY")
+	mixpanel_api_secret   = os.Getenv("MIXPANEL_API_SECRET")
+	redshift_host         = os.Getenv("AWS_REDSHIFT_HOST")
+	redshift_port         = os.Getenv("AWS_REDSHIFT_PORT")
+	redshift_database     = os.Getenv("AWS_REDSHIFT_DB_NAME")
+	redshift_user         = os.Getenv("AWS_REDSHIFT_USER")
+	redshift_password     = os.Getenv("AWS_REDSHIFT_PASSWORD")
```

Comment:
Recommended to exit immediately if env not set. You can use `Get` in https://github.com/segmentio/go-env/blob/master/env.go#L7 and handle error